src/MatrixTypes_Trilinos.f90

```diff
@@ -45,6 +45,8 @@ MODULE MatrixTypes_Trilinos
     INTEGER(SIK) :: ncol
     INTEGER(SIK),ALLOCATABLE :: jloc(:)
     REAL(SRK),ALLOCATABLE :: aloc(:)
+    LOGICAL(SBK) :: setByVal = .FALSE.
```

Comment:
beyond the scope of this, but need doxygen bits in this type definition

Reply:
Added it for all terms.

Reply:
awesome, thanks